benchmarks/source-flotiq/gatsby-browser.js

```diff
@@ -0,0 +1 @@
+import "./styles.css"
```

Comment:
Nit: I would move this to me imported into your layout. It's possible this works fine, I've just never seen this done. I am also wondering if there maybe be some strange side effects happen, if it works at all. 🤷‍♂️

Reply:
Done, thanks for pointing out.